mass_mailing_custom_unsubscribe_event/__manifest__.py

```diff
@@ -1,13 +1,15 @@
 # Copyright 2020 Tecnativa - Pedro M. Baeza
+# Copyright 2020 Tecnativa - João Marques
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 {
     "name": "Allow to unsubscribe discretely from an event",
     "category": "Marketing",
-    "version": "12.0.1.0.1",
+    "version": "13.0.1.0.0",
     "depends": ["event", "mass_mailing_custom_unsubscribe"],
     "author": "Tecnativa, Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/social",
     "data": ["views/event_registration_views.xml"],
+    "demo": ["demo/assets.xml"],
```

Comment:
Lol, contrary to what I told you yesterday, for v13 you don't need to declare this as demo assets. Read below.